### PR TITLE
[PAAR-3560] Remove Jekyll from Flystyles

### PR DESCRIPTION
**Pull request by vicgisso**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -27,6 +27,6 @@ collections:
     output: false
 
 # Assets
-sass:
-  sass_dir:          _scss
-  style:            :compressed
+# sass:
```
Comment (jacarui) [inline _config.yml:30]: Can we remove this?

File: gulp-config/paths.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+var paths = {};
```
Comment (jacarui) [inline gulp-config/paths.js:1]: Is this file still needed?

**Review by jacarui — APPROVED** [review]

**Review by vigosan — APPROVED** [review]
